locales/en.json

```diff
@@ -35,6 +35,8 @@
   "chart": "Chart",
   "holders": "Holders",
   "flippers": "Flippers",
+  "fullscreen": "Fullscreen",
+  "new tab": "Open in New Tab",
```

Comment:
Could you please use Camel case? `newTab`
You do have to add translation for all the other language.

Reply:
Done